backend/src/main/java/codezap/like/service/LikesService.java

```diff
@@ -0,0 +1,44 @@
+package codezap.like.service;
+
+import jakarta.transaction.Transactional;
+
+import org.springframework.stereotype.Service;
+
+import codezap.like.domain.Likes;
+import codezap.like.repository.LikesRepository;
+import codezap.member.domain.Member;
+import codezap.member.dto.MemberDto;
+import codezap.member.repository.MemberRepository;
+import codezap.template.domain.Template;
+import codezap.template.repository.TemplateRepository;
+import lombok.RequiredArgsConstructor;
+
+@Service
+@RequiredArgsConstructor
+public class LikesService {
+
+    private final TemplateRepository templateRepository;
+    private final MemberRepository memberRepository;
+    private final LikesRepository likesRepository;
+
+    public void like(MemberDto memberDto, long templateId) {
+        Template template = templateRepository.fetchById(templateId);
+        Member member = memberRepository.fetchById(memberDto.id());
+        if (likesRepository.existsByTemplateAndMember(template, member)) {
+            return;
+        }
+        Likes likes = template.like(member);
```

Comment:
```suggestion
        Likes likes = new Likes(template, member);
```
`Likes` 생성에 대한 책임을 `Template`에게 넘겨준 이유가 있으신가요? 위와 같이 `LikesSerivce`에서 `Likes`를 생성해도 크게 어색하지 않은 것 같아서요!

Reply:
만약 DB 를 사용하지 않는 콘솔 프로그램이었다면 `Likes` 생성 책임이 누구에게 있는 것이 자연스러울까? 
누구에게 `like()` 라는 메서드가 구현될까? 
정도의 고민에서 나온 코드입니다..!

다들 많이 어색하게 느껴지신다면... 수정하겠습니닷 ㅠ
의견 많이들 달아주세용~